README.md

```diff
@@ -1,2 +1,153 @@
-# colog
-Simple output and logging library for go CLIs.
+# Disgo
+
+Simple output library for go CLIs.
+
+Color provides three essential features for most user-friendly CLI applications:
+
+1. Simple output levels (in `github.com/Ullaakut/disgo/logger`)
+    - Debug outputs are like normal outputs, but are written only on a logger with enabled debug.
+    - Error outputs are written on the error writer, usually stderr, but it's configurable.
```

Comment:
If I got this right, this behavior is optional in our current implementation, maybe we want to change this sentence ?